src/Alert/Alert.stories.tsx

```diff
@@ -25,6 +26,17 @@ export const WithContentAndClose: Story = () => (
   </Alert>
 );
 
+export const WithoutTitle: Story = () => (
+  <Alert close={false} variant="info">
+    <Link>
```

Comment:
This wraps whole content in link, I guess it's not desired?

Reply:
Thanks, for some reason I looked at it and it seemed fine, I need coffee...